Mathlib/CategoryTheory/Monoidal/Category.lean

```diff
@@ -170,12 +213,111 @@ scoped notation "λ_" => leftUnitor
 /-- Notation for the `rightUnitor`: `X ⊗ 𝟙_C ≃ X` -/
 scoped notation "ρ_" => rightUnitor
 
+variable {C : Type u} [𝒞 : Category.{v} C] [MonoidalCategory C]
+
+@[reassoc]
+theorem tensorHom_def' {X₁ Y₁ X₂ Y₂ : C} (f : X₁ ⟶ Y₁) (g: X₂ ⟶ Y₂) :
+    f ⊗ g = X₁ ◁ g ≫ f ▷ Y₂ :=
+  whisker_exchange f g ▸ tensorHom_def f g
+
+/-- Tensor product of identity maps is the identity: `(𝟙 X₁ ⊗ 𝟙 X₂) = 𝟙 (X₁ ⊗ X₂)` -/
+theorem tensor_id (X₁ X₂ : C) : tensorHom (𝟙 X₁) (𝟙 X₂) = 𝟙 (tensorObj X₁ X₂) := by
```

Comment:
Is there a reason that you do not use notation for the tensor product?

Reply:
No, I fixed it.